pkg/lib/session-recording-terminal.jsx

```diff
@@ -0,0 +1,180 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
I think it is better to have this file directly in `pkg/session_recording` and named just `terminal`, if technically possible. Since nothing else will be using this custom terminal component, we don't want to pollute the global namespace of `pkg/lib`.

Also, could you please make a separate commit simply copying this file from `cockpit-components-terminal.jsx`, and then another commit tweaking it for our purposes? This way we'll be able to see what changes you made right in history and in this pull request.